include/openssl/x509v3err.h

```diff
@@ -10,12 +10,6 @@
 
 #ifndef OPENSSL_X509V3ERR_H
 # define OPENSSL_X509V3ERR_H
-# pragma once
-
-# include <openssl/macros.h>
-# ifndef OPENSSL_NO_DEPRECATED_3_0
-#  define HEADER_X509V3ERR_H
-# endif
```

Comment:
Is it a desired change or an accident one?

Reply:
This is autogenerated change with make update , if i remove it travis will fail

Reply:
this file also not for edit

Reply:
Just remove it from your commit.